lib/iris/tests/integration/test_subset.py

```diff
@@ -0,0 +1,56 @@
+# (C) British Crown Copyright 2013 - 2015, Met Office
```

Comment:
I'm still leaning towards deleting this entirely; I don't see that it adds anything compared to the Unit tests.